### ISSUE TITLE
Write/read/compaction of multiple flatbuffers in an object.

### ISSUE THREAD

**Issue by jlefevre** [uccross/skyhook-ceph#4]
Write and read a full object that is composed of a sequence of flatbuffers --faster during initial write-- vs. compacting the existing object's flatbuffer (old rows) with each newly arriving flatbuffer (new rows) --slower during initial write--.

1.  Test the cost during write for each of the two strategies, noting the overhead, so that can inform our cost savings during read given either write strategy.

2. Test the read cost for each of the two strategies.

3.  Similarly, test the cost of compaction for both of the above designs for processing the delete vector for each flatbuffer in an object.  Since this is similar to the second case during initial write, determining the cost tradeoffs will help inform write and compaction strategies.

**Comment by jlefevre** [comment]
Closing this issue, has been moved to new issue #33 for Arrow, and will create a new issue for Flatbuffers only once we have the new Flatbuffer layouts completed (flatbuf-flexbuf, flatbuf-unionRow, flatbuf-unionCol ,...)  This issue also dealt with cost tradeoffs but those will be explored later given each independent layout.
